src/easyscore.js

```diff
@@ -271,12 +292,18 @@ export class EasyScore {
     this.options = Object.assign({
       factory: null,
       builder: null,
+      commitHooks: [
+        setId,
+        setClass,
+        Articulation.easyScoreHook,
```

Comment:
It seems like you're optimizing for the reader, and I'm optimizing for the writer.

I prefer the latter, but in the interest of moving forward, let's leave the defaults in one place, and revisit if this becomes an issue.